package.json

```diff
@@ -17,14 +17,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/origin/origin-web-catalogs.git"
+    "url": "https://github.com/origin/origin-web-catalog.git"
   },
-  "author": "Jeff Phillips, David Taylor",
+  "author": "Red Hat, Inc",
   "license": "MIT",
```

Comment:
@jeff-phillips-18 This should probably be Apache License 2.0 since the rest of OpenShift is

cc @jwforres